components/form/Form.razor.cs

```diff
@@ -202,6 +216,52 @@ private async Task OnInvalidSubmit(EditContext editContext)
             await OnFinishFailed.InvokeAsync(editContext);
         }
 
+        private void RulesModeOnFieldChanged(object sender, FieldChangedEventArgs args)
+        {
+            if (!ValidateMode.IsIn(FormValidateMode.Rules, FormValidateMode.Complex))
+            {
+                return;
+            }
+
+            _rulesValidator.ClearErrors();
```

Comment:
It should work like this(It works fine on my browser):
![image](https://user-images.githubusercontent.com/3509249/117937293-63de8500-b338-11eb-8c43-ec86a94221fb.png)

Something must be wrong, I'ill check it.

Thanks for your review!
I'll refactor the code, because I'am trying to change Rules to configurable, similar to Ant Design React:
![image](https://user-images.githubusercontent.com/3509249/117937808-f54df700-b338-11eb-87f8-93c79eb8fbbb.png)


So I changed PR status to Word In Process.
If possible, please help me review it again after I finish it.

Reply:
Oh, I missed that somehow...sorry to bother you. With demo I used the generated preview. I will test it again once you finish.